homeassistant/components/plex/sensor.py

```diff
@@ -174,6 +179,31 @@ def _update_state_and_attrs(self):
                 libtype=libtype, includeCollections=False
             )
 
+        recent_libtype = LIBRARY_RECENT_LIBTYPE.get(
+            self.library_type, self.library_type
+        )
+        recently_added = self.library_section.recentlyAdded(
+            maxresults=1, libtype=recent_libtype
+        )
+        if recently_added:
+            media = recently_added[0]
+            if recent_libtype == "episode":
+                latest_item = (
+                    f"{media.grandparentTitle} - {media.seasonEpisode} - {media.title}"
```

Comment:
Shouldn’t this be a helper?

Reply:
Ah yes, I had intended to update this to use the helper added in #56312.

Reply:
Updated to use the helper and adjusted the tests. Thanks for catching that. 👍